tests/crud/test_hooks.py

```diff
@@ -39,17 +40,66 @@ async def look_up_existing(row_id: int, values: dict):
     return values
 
 
+async def add_additional_name_details(
+    row_id: int, values: dict, request: Request
+):
+    director = request.query_params.get("director_name", "")
+    values["name"] = values["name"] + f" ({director})"
+    return values
+
+
+async def additional_name_details(row: Movie, request: Request):
+    director = request.query_params.get("director_name", "")
+    row["name"] = f"{row.name} ({director})"
+    return row
+
+
+async def test_raises(row_id: int, request: Request):
```

Comment:
I think the tests are failing due to this.

It's because we use pytest as the test runner, and it thinks that any function starting with `test_` is a unittest. If it's renamed to something like `raises_exception` instead, then it should work.

Reply:
Sorry about that dumb mistake, got it fixed.

Reply:
No worries - thanks!